net.sourceforge.fuse_emulator.Fuse.appdata.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 The Fuse development team -->
+<component type="desktop">
+  <id>net.sourceforge.fuse_emulator.Fuse</id>
+  <replaces>
+    <id>net.sf.fuse_emulator.desktop</id>
+  </replaces>
```

Comment:
can you add the same thing with the provides tag? replaces is newer and not everyone supports it.

Reply:
Ok, @hfiguiere suggested to use `<replaces>`, but I can use `<provides>` instead.

Shall I switch to `<replaces>` in one or two years, or what do you recommend?

Reply:
Use both, not every software store supports replaces.

Reply:
It's in the docs https://docs.flathub.org/docs/for-app-authors/maintenance#renaming-the-flatpak-id